README.md

```diff
@@ -46,7 +46,12 @@ Release
 Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC. 
 Produced at the Lawrence Livermore National Laboratory.
 
-All rights reserved. See details in the file [LICENSE](./LICENSE).
+All rights reserved. 
 
 Unlimited Open Source - BSD Distribution
 `LLNL-CODE-741217` `OCEC-17-187`
+
```

Comment:
Might be good to note 'BSD 3-clause'.

Reply:
@rhornung67 -- are you suggesting that I change it to 

 >  Unlimited Open Source - BSD 3-clause Distribution

Reply:
Yes. Just to be precise.